spec/requests/media/statistics_spec.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+describe '/audio_recordings/:audio_recording_id/original(.:format)', type: :request, aggregate_failures: true do
+  include_context 'shared_test_helpers'
+  create_audio_recordings_hierarchy
+
+  let(:lt_recording) {
+    create(
+      :audio_recording,
+      recorded_date: Time.zone.parse('2019-09-13T00:00:01+1000 '),
+      duration_seconds: audio_file_bar_lt_metadata[:duration_seconds],
+      media_type: audio_file_bar_lt_metadata[:format],
+      original_file_name: Fixtures.bar_lt_file.basename,
+      status: :ready,
+      site: site
+    )
+  }
+
+  before do
+    clear_audio_cache
+
+    # backfill our audio storage with a fixture
+    @test_file = link_original_audio(
+      target: Fixtures.bar_lt_file,
+      uuid: lt_recording.uuid,
+      datetime_with_offset: lt_recording.recorded_date,
+      original_format: audio_file_bar_lt_metadata[:format]
+    )
+
+    Permission.create!(
+      project_id: project.id,
+      user_id: nil,
+      allow_anonymous: true,
+      level: 'reader',
+      creator: owner_user
+    )
+  end
+
+  after(:all) do
+    @test_file&.unlink
+  end
+
+  def expect_stats(
+    user_segment, user_original, user_duration,
+    recording_segment, recording_original, recording_segment_duration
+  )
+    expected = {
+      audio_download_duration: user_duration.nil? ? nil : a_value_within(0.001).of(user_duration.to_d),
+      audio_segment_download_count: user_segment,
+      audio_original_download_count: user_original,
+      original_download_count: recording_original,
+      segment_download_count: recording_segment,
+      segment_download_duration: recording_segment_duration.nil? ? nil : a_value_within(0.001).of(recording_segment_duration.to_d)
+    }
+
+    actual = {}.merge(
+      AudioRecordingStatistics.totals_for(lt_recording),
+      UserStatistics.totals_for(user)
+    )
+
+    expect(actual).to match a_hash_including(expected)
+  end
+
+  def media_url(start_offset, end_offset, format = '.wav')
+    "/audio_recordings/#{lt_recording.id}/media#{format}?start_offset=#{start_offset}&end_offset=#{end_offset}"
+  end
+
+  def original_url
+    "/audio_recordings/#{lt_recording.id}/original"
+  end
+
+  context 'with the admin user', :slow do
```

Comment:
Added:

```
  # these tests are slow because we're actually generating a bunch of media segments. I thought about faking media generation
  # but every fake adds technical debt (see "Why mocking is bad)" and it would be more effort to disable a working service.
```